tests/pass-dep/libc/libc-epoll.rs

```diff
@@ -18,6 +18,7 @@ fn main() {
     test_epoll_ctl_del();
     test_pointer();
     test_two_same_fd_in_same_epoll_instance();
+    test_no_notification_for_unregister_flag();
```

Comment:
This test is "simpler" than test_epoll_ctl_mod, in the sense that test_epoll_ctl_mod also tests that not-registered events do not trigger notifications.
So better test this before test_epoll_ctl_mod, or else it won't ever fail...